### ISSUE TITLE
Institutions: do we want to preserve the contents of 410__9?

### ISSUE THREAD

**Issue by jacquerie** [inspirehep/inspire#204]
Here's the current status of `410__9`:
```
   1 Center of Space Science and Application
   1 Crete Center for Theoretical Physics (CCTP)
   1 Institute of Physical and Chemical Research
   1 Lab. Nucl. En. Phys.
   1 Nat. acad.
   1 National Scientific & Educational Center
   1 Sci. Inst.
   1 Tech
   1 Univ. ctr.
   1 Univ., phys., astron., comput., sci.
   1 acad. si.
   1 http://www.univ-tebessa.dz/fr/
   1 ins., sup. tec. tech.
   1 inst., sci.
   1 univ. ctr. res. cer.comp. mater.
   1 univ., cath.
   1 univ., dept., Sci., comp.
   1 univ., tech., wirt.
   1 web
   4 INSPIRE
   4 Univ.
   7 univ.
  21 ADS
4108 DESY_AFF
6640 DESY
```
I'm not sure what the difference between DESY_AFF and DESY is, but together they comprise the sheer majority of the `name_variants` sources. Is the source still useful, then? Do we want to preserve it?

(There's also some instances of bad data in this field).

**Comment by fschwenn** [comment]
410__9a with 410__9:DESY_AFF are name variants 
410__9a with 410__9:DESY are identifiers (in the past they were used in parallel to 110__u) - I would they say they are obsolete today 

-- 
Florian Schwennsen 
Deutsches Elektronen-Synchrotron DESY 
Building 01 Room O1.446 
phone: +49-40-8998-6190

**Comment by kaplun** [comment]
@fschwenn what is obsolete? 
* the whole 410?
* the whole 410 when $9 is DESY or DESY_AFF?
* the value of 410__9?

**Comment by jacquerie** [comment]
@kaplun, me and @david-caro were discussing this in this exact moment, and your comment appeared : )

**Comment by kaplun** [comment]
Ehehe, when you wrote it in chat I followed the link and I understood what was blocking this issue :dancer:

**Comment by kaplun** [comment]
@fschwenn @jacquerie I don't think we have a conclusive answer of what is obsolete, have we?

**Comment by kaplun** [comment]
Closed by inspirehep/inspire-next#1524
